docs/src/main/tut/docs/cassandra/README.md

```diff
@@ -4,6 +4,265 @@ title: Cassandra
 permalink: /docs/cassandra
 ---
 
-# Cassandra
 
-WIP
\ No newline at end of file
+# Freestyle-Cassandra
+
+[Cassandra] atop **Freestyle** is **`frees-cassandra`**.
+
+Freestyle Cassandra is a purely functional library for interacting with Cassandra supporting both,
+[LowQueryAPI] and [StringQueryInterpolation].
```

Comment:
It will support soon object mapping and other APIs so not sure we need to list those here until they are all complete. I think we should just emphasize that is a Scala Purely Functional driver for Cassandra based on the datastax Java Driver